src/frontend.md

```diff
@@ -221,6 +221,24 @@ export function fakeTicket() {}
 
 1. If some color is repeated several times in code, extract it to theme.
 
+## Disposition of components
+
+- Prefer using local components `(host|mf)/src/components` with a few exceptions
+- Use UI Kit distribution if a component represents a design token (1) [websome](<https://www.figma.com/file/I8WglqziPB5pYlFIqF43JE/Client-UI-Kit-(Web)?node-id=0-1&t=MuFhhOjUXP4sFRr2-0>) or matches with [MUI](https://mui.com/material-ui/)
+- Use MF distribution if a component is required in several repositories and could be shipped as a whole piece with API
```

Comment:
`Use MF distribution` – maybe replace with `Expose component from MF`?

`could be shipped as a whole piece with API` – how about to replace with `represents domain of microfrontend and not being common`?